cfme/services/catalogs/catalog_items/catalog_bundles.py

```diff
@@ -95,9 +95,9 @@ def create(self, name, catalog_items=None, catalog=None, description=None, displ
             'select_catalog': cat_name,
             'select_dialog': dialog
         })
-        if view.basic_info.field_entry_point.value == "":
-            view.basic_info.fill({'field_entry_point': ''})
-            view.basic_info.tree.click_path(
+        if view.basic_info.provisioning_entry_point.value == "":
```

Comment:
@mshriver 
- I am agree with you. This issue occurred because of changes for the entry points in ```BasicInfoForm``` view. Also as you said we have fixed ```AttributeError``` for wrong widget names but need to improve ```create``` method.

- According to my observations,
    - While creating ```catalog bundle```: **Provisioning Entry Point** and **Retirement Entry Point** has default values. So this check ```if view.basic_info.provisioning_entry_point.value == "":``` will always return **false**. Hence we don't need this block of code if there is no such condition where we will get  **Provisioning Entry Point** is **None**.
    - Screenshot of catalog bundle basic info view:
![cat_bundle_entry](https://user-images.githubusercontent.com/25399590/51886525-30b99700-23b6-11e9-9eb9-aba648f849f2.png)